src/nodejs/unit-http/http_server.js

```diff
@@ -480,10 +479,10 @@ Server.prototype.emit_request = function (req, res) {
         this.emit("request", req, res);
     }
 
-    process.nextTick(() => {
+    setTimeout(() => {
```

Comment:
Please explain why this change is important.

Reply:
@mar0x In proxy mode, process.nextTick's callback doesn't execute, and the request not ended before timeout.

Reply:
@mar0x In proxy mode, process.nextTick's callback execute too early, it cause the request not ended until request timeout.
You can simulate the issue by calling "curl http://127.0.0.1:9003/api/", in this case, if we emit finish event in process.nextTick's callback, our request will not ended until request time, but if we change `process.nextTick(cb)` to `setTimeout(cb ,0)`, everything goes fine.